lib/core.ex

```diff
@@ -7,8 +7,18 @@ defmodule Core do
 
   def create_playlist(channel_id) do
     video = Youtube.get_channel_last_video(channel_id)
-    Line.send_text_message("A new video was published #{video.video_title}")
-    create_playlist_from_video(video.video_id)
+
+    case video do
+      {:ok, %{video: video}} ->
+        Line.send_text_message("A new video was published #{video.video_title}")
+        create_playlist_from_video(video.video_id)
+
+      {:ok, :no_video} ->
+        Line.send_text_message("No video was published")
+
+      {:err, %{multiple_videos: video_titles}} ->
```

Comment:
`video_titles` is a list of strings... which will still work here since it happens to be valid [IO data](https://hexdocs.pm/elixir/IO.html#module-io-data), but you might want to use [`map_join`](https://hexdocs.pm/elixir/Enum.html#map_join/3) instead of `map` to define the separator.

```elixir
iex(1)> iodata = ["foo", "bar", "baz"]
["foo", "bar", "baz"]
iex(2)> "hello #{iodata}!"
"hello foobarbaz!"
```